protected/tests/functional/AdminSiteAccessTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Functional test for only logged in as admin user can access Administration Page.
+ *
+ */
+
+class AdminSiteAccessTest extends FunctionalTesting
+{
+    use BrowserSignInSteps;
+    use BrowserPageSteps;
+
+    /**
+     * To test admin user can login and access Administration Page.
+     *
+     * @uses \BrowserSignInSteps::loginToWebSiteWithSessionAndCredentialsThenAssert()
+     * @uses \BrowserPageSteps::visitPageWithSessionAndUrlThenAssertContentHasOrNull()
+     *
+     */
+    public function testItShouldBeDisplayedToUsersWithAdminRole()
+    {
+        $this->loginToWebSiteWithSessionAndCredentialsThenAssert("admin@gigadb.org", "gigadb", "Joe's GigaDB Page");
+        $url = "http://gigadb.dev/site/admin";
+        $this->visitPageWithSessionAndUrlThenAssertContentHasOrNull($url, "Administration Page");
+    }
+
+    /**
+     * To test registered user can login, but cannot access Administration Page.
+     *
+     * @uses \BrowserSignInSteps::loginToWebSiteWithSessionAndCredentialsThenAssert()
+     * @uses \BrowserPageSteps::visitPageWithSessionAndUrlThenAssertContentHasOrNull()
+     *
+     */
+    public function testItShouldNotBeDisplayedToUsersWithUserRole()
+    {
+        $this->loginToWebSiteWithSessionAndCredentialsThenAssert("user@gigadb.org", "gigadb", "John's GigaDB Page");
+        $url = "http://gigadb.dev/site/admin";
+        $this->visitPageWithSessionAndUrlThenAssertContentHasOrNull($url, "Error 403");
```

Comment:
Based on Rija's comment re: cost of instantiating variables, the `$url` variable can be dropped and replaced with the `http://gigadb.dev/site/admin` link string.

Reply:
`$url` has been dropped.